### PR TITLE
Beginnings of an api to handle access to servers

### PR DESCRIPTION
**Pull request by mpdavis**
Played around with ansible and came up with how we can handle the api.  This is a very simple example, but it is sufficient to show we will be able to do what we need out of ansible.

From this we can start constructing the API.

Please review
@organman91 
@computmaxer  

image

### CODE REVIEW

**Comment by qinxiaoapp** [comment]
+1 !! Nice work and thx for putting time into it.

File: src/vinz/__init__.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    Simple test to ping all of the hosts.
+    """
+
+    # Eventually, this will be need to be pulled out of the database.
+    # Just need to make a list of hosts.  DNS names and IPv4 addresses.  No IPv6.
+    hosts = ['test.example.com']
+
+    inventory = Inventory(hosts)
+
+    # TODO: We need to find a way to handle the private key file.
```
Comment (computmaxer) [inline src/vinz/__init__.py:17]: At first thought, I'm thinking it should be a setting somewhere (settings.py, or some config file that we read in on startup that gets generated during installation)

`PATH_TO_GLOBAL_PRIVATE_KEY_FILE = '/var/srv/vinz/keys/id_rsa'`

**Comment by computmaxer** [comment]
Looks great!  Glad it is easy. :+1:

**Comment by organman91** [comment]
:thumbsup:

**Pull request merged**